example/src/components/Carousel6.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Carousel from 'react-gallery-carousel';
+import 'react-gallery-carousel/dist/index.css';
+
+const Carousel6 = ({ images }) => {
+  const thumbnails = images.map((_, idx) => (
```

Comment:
It will look like this.
![SS 2021-10-04 at 00 38 29](https://user-images.githubusercontent.com/49315663/135756046-f427b790-8b9a-4003-a7f9-f4b05fd5ca93.jpg)